mne/realtime/tests/test_fieldtrip_client.py

```diff
@@ -116,7 +116,7 @@ def test_fieldtrip_rtepochs(free_tcp_port, tmpdir):
                                                   int(ev.comment))
 
                 _call_base_epochs_public_api(epochs_rt, tmpdir)
-                epochs_rt.stop()
+                epochs_rt.stop(stop_receive_thread=True)
```

Comment:
shouldn't this be made `True` by default?

Reply:
Makes sense, I changed the default in ``RtEpochs.stop`` (I've left the parameter in the test to be explicit).